Probesys/bileto
# imp: Automate the assignment of teams to tickets

## Related issue(s)



https://github.com/Probesys/bileto/issues/647

## How to test manually



1. Create two "responsible" teams and give them an authorization on the same organization
2. Create a ticket in this organization and verify that the team is automatically selected
3. Edit the organization (settings) to declare the other team as the responsible team
4. Create a ticket and verify that the other team is automatically selected
5. Remove the authorization from the responsible team
6. Verify that the organization responsible team is now set to "Automatic"

## Checklist
